freemindlatex/static_files/report.tex

```diff
@@ -0,0 +1,51 @@
+\documentclass[CJK]{article}
```

Comment:
The current formatting is too latex-ish, not pretty. Try to use narrower margins, change the default font, just to make the doc prettier.